src/app/gui/coins/coins.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Input, OnInit } from "@angular/core";
+import { CurrencyComponent } from "../currency/currency.component";
+
+@Component({
+  selector: "lux-coins",
+  templateUrl: "../currency/currency.component.html",
+  styleUrls: ["../currency/currency.component.css"]
+})
+export class CoinsComponent extends CurrencyComponent implements OnInit {
+  @Input() count: number;
+  image: string = "/lu-res/ui/ingame/achievements_i4.png";
```

Comment:
That image is double the height of the visible portion, which causes a large gap between the lines on the items component page. Can you cut that down a bit with CSS?